Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -366,7 +369,17 @@ void ProjectData::parseProcesses(BaseLib::ConfigTree const& processes_config,
         }
         else
 #endif
-#ifdef OGS_BUILD_PROCESS_HYDROMECHANICS
+#ifdef OGS_BUILD_PROCESS_HEATTRANSPORTBHE
+            if (type == "HEAT_TRANSPORT_BHE")
+        {
+            process = ProcessLib::HeatTransportBHE::createHeatTransportBHEProcess(
+                *_mesh_vec[0], std::move(jacobian_assembler),
+                _process_variables, _parameters, integration_order,
+                process_config, _curves);
+        }
+        else
+#endif
+#ifdef OGS_BUILD_PROCESS_HYDRO_MECHANICS
```

Comment:
Don't change the macro. See line 59 above or 56 in original code.

Reply:
fixed